packages/node-core/src/indexer/fetch.service.ts

```diff
@@ -102,10 +102,21 @@ export abstract class BaseFetchService<DS extends BaseDataSource, B extends IBlo
 
     await Promise.all([this.getFinalizedBlockHead(), this.getBestBlockHead()]);
 
+    this.projectService.startHeight;
+
     if (startHeight > this.latestHeight()) {
-      throw new Error(
-        `The startBlock of dataSources in your project manifest (${startHeight}) is higher than the current chain height (${this.latestHeight()}). Please adjust your startBlock to be less that the current chain height.`
-      );
+      // This is at init stage, lastProcessedHeight should be always - 1 from the startHeight in this case
+      // this is reverse calculated from projectService.nextProcessHeight()
```

Comment:
It seems like the source of startHeight needs to be fixed rather than adding special cases here